src/main/java/org/sefglobal/scholarx/model/Email.java

```diff
@@ -0,0 +1,35 @@
+package org.sefglobal.scholarx.model;
+
+import lombok.Data;
+
+@Data
+public class Email {
+    private String email;
```

Comment:
```suggestion
    private String emailAddress;
```
Make the var meaningful